### PR TITLE
add list function and some part of sync function

### PR DESCRIPTION
**Pull request by W-LT**

### CODE REVIEW

File: deadline_fighters/desktop-client/index.html [hunk 1]
```diff
@@ -59,6 +70,148 @@
 			}
 		}, false);
 		
+	
+//creat folder
```
Comment (sranjani-s) [inline deadline_fighters/desktop-client/index.html]: create -> spelling error

File: deadline_fighters/desktop-client/index.html [hunk 2]
```diff
@@ -59,6 +70,148 @@
...
+bucket.listObjects({}, function(err, data) {
+    if (err) console.log(err, err.stack); // an error occurred
+    $('#objectList').empty();   //empty the list
+    var objects = data.Contents;
+	  
+    $.each(objects, function(i, content) {
+       //successful response in console
+      console.log(data);  
+       //list the file name to the page
+	  $('#objectList').append("<button  id="+i+" value= '"+content.Key+"'>"+content.Key+"</button><br>");
```
Comment (sranjani-s) [inline deadline_fighters/desktop-client/index.html]: Alignment issue

File: deadline_fighters/desktop-client/index.html [hunk 3]
```diff
@@ -59,6 +70,148 @@
...
+       return console.error(err);
+   }
+   console.log("folder created successfully.");
+});	
+				
+		
+		
+//list
+bucket.listObjects({}, function(err, data) {
+    if (err) console.log(err, err.stack); // an error occurred
```
Comment (sranjani-s) [inline deadline_fighters/desktop-client/index.html]: Put the statements under if inside brackets and in a different line for easier reading.

File: deadline_fighters/desktop-client/index.html [hunk 4]
```diff
@@ -59,6 +70,148 @@
...
+	  
+	  
+//sync——only can download all files now, the function of upload all files are not implemented.
+$(sync).click(function(){
+var fs = require("fs"); 
+var bucket2 = 'deadlinefighters';		  
+fs.readdir('D:\\deadlinefighters\\',function(err,files){
+	console.log(files);
+	var s3_upload = new AWS.S3();
+	for(var i=0;i<files.length; i++){
```
Comment (sranjani-s) [inline deadline_fighters/desktop-client/index.html]: Pre-incrementer is faster than post-incrementer.

File: deadline_fighters/desktop-client/index.html [hunk 5]
```diff
@@ -59,6 +70,148 @@
...
+       //list the file name to the page
+	  $('#objectList').append("<button  id="+i+" value= '"+content.Key+"'>"+content.Key+"</button><br>");
+	
+}); 
+	    
+	  
+	  
+//sync——only can download all files now, the function of upload all files are not implemented.
+$(sync).click(function(){
+var fs = require("fs");
```
Comment (sranjani-s) [inline deadline_fighters/desktop-client/index.html]: Is this required under every function? Can't we put it inside a global variable?

File: deadline_fighters/desktop-client/index.html [hunk 6]
```diff
@@ -59,6 +70,148 @@
...
+    var objects = data.Contents;	  
+    $.each(objects, function(i, content) {
+	  	  var fs = require('fs');	
+		  var file = content.Key;
+		  console.log(file);  
+		  var filePath = '/deadlinefighters'+'/' + file;
+		  console.log(filePath); 
+		  var bucket1 = 'deadlinefighters';
+		  var s3_down_load = new AWS.S3();
+		  var s3_dele_te = new AWS.S3();
```
Comment (sranjani-s) [inline deadline_fighters/desktop-client/index.html]: Please use more appropriate variable names. How exactly is dele_te and down_load different from delete and download?

File: deadline_fighters/desktop-client/index.html [hunk 7]
```diff
@@ -59,6 +70,148 @@
...
+        fs.writeFileSync(filePath, data.Body)
+		console.log("download sucessfully");
+		}
+    });
+		  });
+	
+}); 
+		  		 
+
+			  	  });
```
Comment (sranjani-s) [inline deadline_fighters/desktop-client/index.html]: Alignment issues.

File: deadline_fighters/desktop-client/index.html (hunk 5 above)
Comment (W-LT) [reply deadline_fighters/desktop-client/index.html]: I will make the change to a different pull request

**Comment by W-LT** [comment]
Raised another pr to resolve conflicts

**Pull request closed without merging**